scripts/setup.sh

```diff
@@ -6,17 +6,28 @@
 #
 # Commands that may fail have "|| return" to avoid continuing or interfering with terminal.
 
-if [ ! -z $NVM_DIR ]; then # skip nvm steps if not available
-  echo 'Configuring node 12...'
-  source ~/.nvm/nvm.sh
-  nvm install || return
+# Install yarn if it does not exist.
+if ! which yarn > /dev/null; then
+  echo 'yarn is required for setup, installing...'
+  if ! which brew > /dev/null; then
```

Comment:
On second thought, I am proposing removing this entirely. The script [assumes](https://github.com/artsy/metaphysics/blob/3c79914b378e75b040a8074919070c6148835a03/scripts/setup.sh#L1) you have pre-reqs from [potential setup](https://github.com/artsy/potential/blob/main/scripts/setup) (which installs brew) and some contributors may want to install `yarn` by other means. So I think stopping the script here is redundant considering if brew is missing L16 will blow up and let the using know that `brew: command not found`. This will be more consistent with `memcached` install step below.